Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityRegisteredServiceProvidersProfile.asset

```diff
@@ -19,7 +19,7 @@ MonoBehaviour:
         Microsoft.MixedReality.Toolkit
     componentName: Unity Joystick Manager
     priority: 10
-    runtimePlatform: -1
+    runtimePlatform: 31
```

Comment:
Not sure this is a good change. Unity Joysticks should work on any platform

Reply:
If not done this way the UnityJoystick will be picked, not the OculusAndroidDeviceManager I created for Android platform conciderations. I did not want to add the Oculus controller into the existing Manager so I added mine in 8th position, but to be reached, this has to be done.

Reply:
Why create a controller that already handles this case?

Wouldn't it make more sense to have the generic unity joystick support this instead?

Reply:
I'm not sure. So far it only detects XBox controllers. the derived OpenVRDeviceManager detects all OpenVR controllers but only works for standalone. WindowsMixedRealityDeviceManager manages the WindowsMixedRealityController. This looks to me as if the UnityJoystickManager would be the best bet, but it does not make real sense either as every SDK (OpenVR, UWP ...kind of) has its manager.
If it follows a certain logic, why not .)

Reply:
what is really confusing is, to even get to the OpenVR device manager, the UnityJoystickManager gas to actively handle controllers with OpenVR in their names and revoke those